chainermn/communicators/_communication_utility.py

```diff
@@ -183,3 +185,17 @@ def _get_nccl_type_id(dtype):
     else:
         raise ValueError(
             'dtype must be float16, float32, or float64.')
+
+
+def get_allreduce_grad_dtype(allreduce_grad_dtype=None):
```

Comment:
where is  `get_allreduce_grad_dtype` used?

Reply:
I created the function during the development but i's no longer necessary any more. I will remove it. thanks